tests/e2e/utils/src/page-utils.js

```diff
@@ -197,18 +198,35 @@ const evalAndClick = async ( selector ) => {
 };
 
 /**
- * Select a value from select2 input field.
+ * Select a value from select2 search input field.
  *
  * @param {string} value Value of what to be selected
- * @param {string} selector Selector of the select2
+ * @param {string} selector Selector of the select2 search field
  */
 const selectOptionInSelect2 = async ( value, selector = 'input.select2-search__field' ) => {
 	await page.waitForSelector(selector);
+	await page.click(selector);
 	await page.type(selector, value);
 	await page.waitFor(2000); // to avoid flakyness, must wait before pressing Enter
 	await page.keyboard.press('Enter');
 };
 
+/**
+ * Search by any term for an order
+ *
+ * @param {string} value Value to be entered into the search field
+ * @param {string} orderId Order ID
+ * @param {string} customerName Customer's full name attached to order ID.
+ */
+const searchForOrder = async (value, orderId, customerName) => {
```

Comment:
I really like how you broke this out into a helper function and used it in the tests 👍